### ISSUE TITLE
Step 2 Typography Audit: Audit DS typography PR for product impact

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18100]
6/5 Unblocked: https://dsva.slack.com/archives/C06V7AAFVH7/p1717618639498089

## Description
The Design system is updating the typography sitewide. This will be done in 2 steps. This is the audit for Step 2 - Increase base font-size from 16px to 16.96px - Wednesday June 5th

- The ask: Again visual QA starting on May 22nd and lasting one sprint.
- IMPORTANT: Focus on reviewing any "imposter", or exempted v1, components. No development is required, only QA. The base font size increase should not break imposter or v1 components since those will inherit both the new base font size and also the scaled rem values. However, given that they are not Design System web-components the Design System team will not have been able to QA them in advance.
- NOTE: v3 web-components have already moved to 16.96px within the components and you will have already moved to v3 by now.
- This change increase our base font size to match the normalized value of 16.96px for Source Sans Pro in the USWDS.

## Steps for Implementation
Review product portfolio in the Design system PR for visual issues

## Environment
http://8d73883f3b28cbaa19f14bc69fc6c744.review.vetsgov-internal/

 ## QA spreadsheet 

[ Google Spreadsheet](https://docs.google.com/spreadsheets/d/1rEKHNoeC4x6jW4e8mgYZL991m781CgblGXNS7iJzgU8/edit?usp=sharing)
 
### Facilities products
- [x] VAMC System
- [x] VAMC Health Services
- [x] VAMC Locations List
- [x] VAMC Facility
- [x] VAMC Operating status
- [x] VAMC Billing & Insurance
- [x] VAMC Medical Records office
- [x] VAMC Register for Care
- [x] VAMC Policies
- [x] VAMC VA Police
- [x] VAMC Detail page
- [x] VAMC System Banners
- [x] Leadership List & Staff pages
- [x] News Releases & Lists
- [x] Stories and Lists
- [x] Vet Center
- [x] Vet Center Locations
- [ ] VBA (Reviewing in staging post launch)
- [x] Facility Locator
- [x] Facility Locator Detail pages

### Public Websites products
- [x] Header / Footer
- [ ] Injected Header / footer (Engineer will need to review)
- [x] Homepage
- [x] Veterans Crisis Line modal
- [x] Global Search results
- [x] Benefits Detail Page 
- [x] Benefits Hub Landing Page 
- [x] Campaign Landing Page 
- [x] Events & List
- [x] Resources and Support
- [x] FAQ page 
- [x] Reusable Q&A 
- [x] Step- by-Step
- [x] Find a Form 
- [x] Outreach Hub
- [ ] Income Limits (Engineer will need to review, needs to be tested locally)
- [x] Discharge upgrade wizard
- [x] PACT Act
- [x] CTA widgets
- [x] BTSSS Widget
- [x] Full width banner
- [ ] Promo banner (Reviewing in staging post launch)

### Other
- [x] All products/templates are reviewed by Design
- [x] Any issues are documented within the QA spreadsheet
- [ ] Ticket are created in GitHub and reviewed with Product to determine next steps

**Comment by davidmpickett** [comment]
Removing sprint 4 since we are blocked

**Comment by jilladams** [comment]
Removing Blocked label. DST has delivered info! 
https://dsva.slack.com/archives/C06V7AAFVH7/p1717618639498089
For more information [checkout the #platform-design-system canvas](https://dsva.slack.com/canvas/C01DBGX4P45?focus_section_id=temp:C:CVT1d5718ee7e8b4162af47bb741)

- **When**: June 6th - 25th
  - June 6th - June 12th: initial QA window (review instance will remain unchanged)
  - June 13th - 17th: DST to address any feedback, update review instance
  - June 18th - 25th: Final review window
  - June 26th: [PR to be merged](https://github.com/department-of-veterans-affairs/vets-website/pull/30089).
- **What**: Normalization for all typography elements (paragraphs, labels, lists, links) using source sans pro to 16.96px
- **Impact**: 0.96px increase to elements listed above on [VA.gov](http://va.gov/)
- **The ask**: Similar to the first typography PR, we would like teams to access their applications using [our review instance](http://d8405876e5ef9883fc31154f07802640.review.vetsgov-internal/) and make sure content flow looks normal.
  - If everything looks good, approve the PR
  - If something needs to be addressed, **please leave a review comment on the** [package.json file](https://github.com/department-of-veterans-affairs/vets-website/pull/30089/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R261), stating what team you are from, a screenshot of the content that needs to be addressed, and a link to the manifest.json file of your application in vets-website. **Please thread any additional comments to your original top level comment on the package.json file.**

**Comment by jilladams** [comment]
Talked to Laura live. This ticket is done with initial testing. Any findings are added to #18285 for @chriskim2311 to identify where things are coming from in the code. This ticket is closeable.

**Comment by laflannery** [comment]
@jilladams I was reminded today that we have to go back in and retest the issues that we found between the 18th and the 25th. At scrum today I asked Michael if it would make more sense to create a new ticket for this review or just repoen this original ticket and keep it all contained. He said to reopen so I'm going to do that and tag this into Sprint 6 but let us know if you feel differently for reporting purposes

**Comment by jilladams** [comment]
@laflannery @thejordanwood I think I saw in Slack today that we confirmed the changes and we feel done here. Am I wrong?

**Comment by laflannery** [comment]
@jilladams I have a comment to Jami in the PR. We might need to re-review once she answers that but it should be very brief because there weren’t many issues left

**Comment by laflannery** [comment]
Ok, closing for real this time. I just re-reviewed and confirmed the last of the issues we reported to DST and everything looks good.

We have found some things that are a bit out of scope that we will need to address, regarding fonts not resizing properly when the user sets the browser font size to something other than the standard. I will be adding issues/discussing how best to address this
